HelpSource/Classes/SynthDef.schelp

```diff
@@ -443,3 +446,35 @@ Pbind(
 )
 ::
 
+
+subsection:: Inline ControlSpec definitions
+Sometimes, it can be clearer to specify a link::Classes/ControlSpec:: for a parameter at the point of it's usage in a SynthDef, rather than elsewhere.
+Specs can be set either when contructing a link::Classes/NamedControl:: or as a property of a SynthDef argument.
+code::
+(
+~def = SynthDef(\argumentExample, {
+	|freq, amp|
```

Comment:
using a new line for args has its benefits, but is rather unusual. Maybe you can just make it less standing out for the eye if you keep with the usual convention.